### ISSUE TITLE
[FYI] Project moved to GitHub Organization

### ISSUE THREAD

**Issue by raxod502** [radian-software/riju#161]
This issue is just a public explanation of why Riju moved to a GitHub Organization, and what the implications are. I should have posted this earlier, but the honest reason why I didn't is I was in the middle of moving apartments and I just forgot! (I did announce it other places, though, like https://github.com/radian-software/straight.el/issues/946.)

Why I did it:
* Donations for Riju were going to my personal bank account, and to be more honest and transparent about how I was spending those donations, I wanted to use a separate business account instead. ([financial records from my personal spending](https://docs.google.com/spreadsheets/d/15Us9KLXaJ6B1lNhrM6GV6JmmeKqNc8NNeTnaWiAhozw/edit#gid=1451084601); [financial records from my business account](https://docs.google.com/spreadsheets/u/3/d/e/2PACX-1vSUB8vgGFVeLyOL5-R64DvxmEAbSq2_ffFOEU_rEAYl0Q0S_DN6ef3NYrnsNZGm1I1ngikmvzzr-sUE/pubhtml))
* GitHub Organizations have a lot more options than personal accounts; in particular, it's now possible for me to grant a lot more access to collaborators on my various open-source projects than it was before.
* In the United States, anyone can wreck your life with a malicious lawsuit if they have enough money to spend on it. This is something [I've been threatened with in the past](https://intuitiveexplanations.com/tech/replit/), and it sucked. My lawyer advised me that one of the best ways to avoid something like that happening again is to conduct my open-source work as part of an LLC that I set up for this purpose, so this is me following that advice.

What actually changed:
* The GitHub repositories moved from https://github.com/raxod502 to https://github.com/radian-software (but there are redirects from the old locations, so existing tooling should continue to work).
* The license notices updated their copyright statements to say "Radian LLC and contributors" instead of "Radon Rosborough" (but they are all still under MIT licensing).
* The contact email addresses changed from radon.neon@gmail.com to contact@radian.codes (but I still read both of them).
* All of my infrastructure for these projects (AWS, Heroku, Fathom, etc.) moved to separate accounts specifically for Radian LLC, with billing from a separate business account (but the same things are running on them).
* I created separate donation accounts (GitHub Sponsors, Patreon, PayPal, etc.) for Radian LLC and got rid of the ones that go to me personally.

More FAQ available at https://radian.codes/.

On May 12, I sent out this message to my GitHub Sponsors (thank you, @JoshuaTheMiller @JasonSteving99 @plondon @romatthe @shanesveller!) explaining that any future donations would need to be re-routed:

> TLDR: Please move your sponsorship from https://github.com/sponsors/raxod502 to https://github.com/sponsors/radian-software, it will help me keep making projects like [Riju](https://github.com/radian-software/riju)!
> 
> Hey friends,
> 
> I'm finally moving my projects to a GitHub organization [**@radian-software**](https://github.com/radian-software) rather than my personal account [**@raxod502**](https://github.com/raxod502). This does a couple of things:
> 
> Makes it easier to collaborate with other open-source developers on projects like [`straight.el`](https://github.com/raxod502/straight.el) and [Selectrum](https://github.com/raxod502/selectrum), since GitHub limits many of its collaboration features to organizations.
> Lets me keep the finances for donations totally separate from my personal bank account, so that I can be extra sure I'm not accidentally using the money on something unrelated to these open-source projects. (All my open-source finances will be publicly available [on GitHub](https://github.com/radian-software/financials) going forward, as they have been already for Riju.)
> Allows me to conduct open-source work as part of a [limited liability company](https://radian.codes/) that I formed, which will let me be less scared of people who want to [use their money to shut down open-source work](https://intuitiveexplanations.com/tech/replit/), if heaven forbid that kind of situation should come up again. Frankly I would have preferred to just not deal with the paperwork, but after the Replit situation, a lawyer told me this is what you're supposed to do as someone who puts out a lot of open-source work, even if you're not interested in starting a company.
> Anyway, to make this happen, what I'm asking from you is a simple bookkeeping change: to move your existing sponsorship from https://github.com/sponsors/raxod502 to https://github.com/sponsors/radian-software at your convenience.
> 
> In case you're curious, for the foreseeable future, the vast majority of donations will go to [Riju](https://github.com/radian-software/riju), as they do currently. I've also put some other pieces of open-source infrastructure under the radian-software organization, like [GNU ELPA Mirror](https://github.com/radian-software/gnu-elpa-mirror) and [Tidier](https://github.com/radian-software/tidier). There are some other pieces of overhead, like domain name fees from Namecheap, a separate email domain, and some government fees for running a company, but I'm trying to keep those minimal compared to infrastructure costs. In any case, everything will be publicly reported [on GitHub](https://github.com/radian-software/financials).
> 
> One new project I'm working on is a miniature educational clone of the Heroku platform, deployed on Kubernetes. It's a bit in the same vein as Riju, but with a different focus. Like Riju was an experiment in learning about containerization technology, this new project is an experiment in learning about Kubernetes and container orchestration at a larger scale. If it works out, I'll make it a public thing people can register accounts on and deploy their own web applications onto.
> 
> If you have any questions about this change, any of the projects I'm working on, or anything else, feel free to reach out! I'm always happy to chat.
> 
> Best,
> Radon

My personal account has not had any hosting expenses since early May, and everything since then has been paid out of my business account. I've submitted a ticket to GitHub Support today asking for them to disable my personal sponsor account, and ([as promised](https://github.com/radian-software/riju/commit/ff95d0aadb8f7b7af523fdcfd65972842122120e)) have donated the [leftover proceeds](https://docs.google.com/spreadsheets/d/15Us9KLXaJ6B1lNhrM6GV6JmmeKqNc8NNeTnaWiAhozw/edit#gid=1451084601) from my personal account to the EFF.

![image](https://user-images.githubusercontent.com/6559064/181921098-6ca37350-07b0-4ce1-8448-3531be51d776.png)

**Comment by JoshuaTheMiller** [comment]
My sponsorship has finally been updated 😅 Keep up the good work!

**Comment by raxod502** [comment]
Closing the issue as I think ample time has been given to update community members on the reasoning behind the repository move.
